blocktx/peer_handler_test.go

```diff
@@ -388,7 +386,9 @@ func TestFillGaps(t *testing.T) {
 			}
 
 			logger := slog.New(slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{Level: slog.LevelInfo}))
-			peerHandler := NewPeerHandler(logger, storeMock, bockChannel, 100)
+			batchSize := 4
```

Comment:
why 4? Can we have it as const somewhere above?

Reply:
The batch size of 4 was there before and as it is a unit test I didn't create a separate const for it.

It is 4 in this case so that the test will also cover this if clause: https://github.com/bitcoin-sv/arc/blob/441a5793da093f357732cfc8549e65a33ef31394/blocktx/peer_handler.go#L507